src/functions/modalSubmitGiveaway.js

```diff
@@ -0,0 +1,104 @@
+const { MessageEmbed } = require('discord.js');
+
+function createGiveawayEmbed(description, timelimit, emoji, image, winner) {
+    let standardEmbedFields = [{
+        name: '🎁 Item',
+        value: description
+    },
+    {
+        name: '➡️ How to enter',
+        value: 'React to this message with ' + emoji
+    },
+    {
+        name: '🏆 Winners',
+        value: 'Winners are picked at random, and prizes are sent automatically through a direct message here on Discord when the giveaway concludes. Make sure you allow direct messages from other members!'
+    }]
+    if (winner != '') {
+        let embed = new MessageEmbed()
+            .setColor('GREEN')
+            .addFields(
+                standardEmbedFields[0],
+                standardEmbedFields[1],
+                standardEmbedFields[2],
+                {
+                    name: '🏁 Giveaway ended',
+                    value: 'Winner: ' + winner
+                }
+            )
+            .setImage(image)
+            .setFooter({text: `Giveaway ended`});
+        return embed;
+    }
+    let embed = new MessageEmbed()
+            .setColor('YELLOW')
+            .addFields(
+                standardEmbedFields[0],
+                standardEmbedFields[1],
+                standardEmbedFields[2]
+            )
+            .setImage(image)
+            .setFooter({text:`Ending in ${Math.ceil(timelimit/60)} ${Math.ceil(timelimit/60) < 2?'hour':'hours'}`});
+    return embed;
+};
+
+async function modalSubmitGiveaway (interaction) {
+    //const channel = '838711380076396574';
+    const channel = '975791268741406800';
```

Comment:
Channel ID should be configurable and not hardcoded. You can put this in the .env file for now until we have implemented a proper config system.